src/js/media-container.js

```diff
@@ -66,8 +73,10 @@ template.innerHTML = `
       width: 100%;
     }
   </style>
-  <slot name="media"></slot>
-  <slot></slot>
+  <section>
```

Comment:
Why add section here? (I think I just removed an internal container a few PRs ago) The downside of having an internal container is  if you want to change the flex-direction of the media-container, you can't easily when the setting is on a shadow dom element. I needed that feature in the spotify audio player demo.

Reply:
This is also for applying descriptive landmarks for a11y (unfortunately we cannot assign the role to the custom element itself). Let me know if you'd like to touch base on this. Otherwise, I'll plan on ensuring that the spotify example works/lays out with the new DOM structure.